content/sensu-go/5.15/getting-started/enterprise.md

```diff
@@ -10,12 +10,13 @@ menu:
     parent: getting-started
 ---
 
-Sensu Go offers license-activated features designed for monitoring at scale.
+Sensu Go offers commercial features designed for monitoring at scale.
 [Contact the Sensu sales team][1] for a personalized demo and free trial.
-License-activated features are available for all Sensu Go packages and downloads.
-See the [products page][3] for a complete feature comparison.
+Commercial features are free for your first 100 entities.
+For more than 100 entities, [commercial features][3] are available for all Sensu Go packages and downloads.
```

Comment:
These 2 sentences are a little confusing together. Basically, commerical features are free for your first 100 entities, _and_ for whatever entities go past the 100 entity limit 😅 

I think something more concise like `Commercial features are available for free in all Sensu Go packages and downloads.` If we'd like to add anything about functionality past the limit, it should be with regard to creating events/entities etc.

Reply:
What about this (to align with the info in the release notes):

All commercial features are available for free in the packaged Sensu Go distribution up to an entity limit of 100.

Reply:
That should work!